govcd/access_control_catalog_test.go

```diff
@@ -0,0 +1,339 @@
+// +build functional catalog ALL
+
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// GetId completes the implementation of interface accessControlType
+func (catalog Catalog) GetId() string {
+	return catalog.Catalog.ID
+}
+
+// GetId completes the implementation of interface accessControlType
+func (catalog AdminCatalog) GetId() string {
+	return catalog.AdminCatalog.ID
+}
+
+func (vcd *TestVCD) Test_AdminCatalogAccessControl(check *C) {
+	if vcd.config.VCD.Org == "" {
+		check.Skip("Test_AdminCatalogAccessControl: Org name not given.")
+		return
+	}
+	org, err := vcd.client.GetOrgByName(vcd.config.VCD.Org)
+	check.Assert(err, IsNil)
+	check.Assert(org, NotNil)
+	adminorg, err := vcd.client.GetAdminOrgByName(vcd.config.VCD.Org)
+
+	check.Assert(err, IsNil)
+	check.Assert(org, NotNil)
+	catalogName := "ac-admin-catalog"
+	// Create a new catalog
+	adminCatalog, err := adminorg.CreateCatalog(catalogName, catalogName)
+	check.Assert(err, IsNil)
+	check.Assert(adminCatalog, NotNil)
+	AddToCleanupList(catalogName, "catalog", vcd.config.VCD.Org, "Test_CatalogControlAccess")
```

Comment:
Invalid test name reference. 
```suggestion
AddToCleanupList(catalogName, "catalog", vcd.config.VCD.Org, check.TestName())
```

Reply:
fixed